rcloud.support/R/rcloud.support.R

```diff
@@ -495,8 +495,14 @@ rcloud.is.notebook.visible <- function(id)
     visibility
   }
 
-rcloud.set.notebook.visibility <- function(id, value)
+rcloud.set.notebook.visibility <- function(id, value){
   rcloud.set.notebook.property(id, "visible", value != 0);
+  if(value){
+    response <- update.solr(rcloud.get.notebook(id),rcloud.notebook.star.count(id))
```

Comment:
Should we check for binary assets in the update function and prune these assets out from checking in ? This would have never worked ever since the solr schema has a data type of string. 
Or we go with the base64encoded string ?